src/components/internals/create-select-styles.js

```diff
@@ -20,12 +20,14 @@ const controlStyles = (props, theme) => (base, state) => {
   return {
     ...base,
     fontSize: overwrittenVars[designTokens.fontSizeForInput],
-    backgroundColor: state.isDisabled
+    backgroundColor: props.isDisabled
```

Comment:
We're now using `props.isDisabled` instead of the internal `state.isDisabled`, because `isReadOnly` is also making the SelectInput become internally disabled (see [this line](https://github.com/commercetools/ui-kit/pull/1192/files#diff-8dd7f63a278e2dc6ae659f5036bafbb3R64)), but we can check the props to differentiate between the two modes for the styles.